app/src/components/layout/Header.tsx

```diff
@@ -144,20 +161,28 @@ const Header: React.FC = () => {
                 </picture>
                 Species Inventory Management System
               </Link>
-              <LoggedInUser />
+              {(!keycloakWrapper?.keycloak?.authenticated || !keycloakWrapper?.hasLoadedAllUserInfo) && (
```

Comment:
Can be done later: We'll probably want to add a utility function (`isUserAuthenticated` or something) that checks if the user is authenticated and loaded so we dont have to past all of this everywhere.

`

Reply:
sounds good, will look into in the next iteration of this :)